src/local/butler/lint.py

```diff
@@ -34,23 +164,32 @@ def execute(_):
   else:
     _, output = common.execute('git diff --name-only FETCH_HEAD')
 
+  file_paths = [f for f in output.splitlines() if os.path.exists(f)]
   py_changed_file_paths = [
-      f for f in output.splitlines() if f.endswith('.py') and
+      f for f in file_paths if f.endswith('.py') and
       # Exclude auto-generated files.
       not f.endswith('_pb2.py') and not f.endswith('_pb2_grpc.py')
   ]
-  go_changed_file_paths = [f for f in output.splitlines() if f.endswith('.go')]
+  go_changed_file_paths = [f for f in file_paths if f.endswith('.go')]
+  yaml_changed_file_paths = [f for f in file_paths if f.endswith('.yaml')]
 
   for file_path in py_changed_file_paths:
-    if os.path.exists(file_path):
-      common.execute('pylint ' + file_path)
-      common.execute('yapf -d ' + file_path)
+    common.execute('pylint ' + file_path)
```

Comment:
yes probably another cl to account for this.

Reply:
>problem is output becomes more messed up to read if you combine everything 

Why? They aren't printing in parallel, just print one error after another, like a compiler does.

>once you have a failure, person would anyway have to run "python butler.py lint" and then fix failure one at a time and only once it finishes without error, then reupload.

The same is true for compilers, but they go to great effort to continue printing more error messages after the first error, because it is easier to fix all issues at once than to rerun the command and fix them one at a time.

When doing the mutator plugin CL it was very annoying to run the command for each error (particularly since the lint command is slow when there are many files to run on).